lib/commands/context.js

```diff
@@ -192,8 +194,11 @@ async function setupExistingChromedriver (chromedriver) {
 }
 
 async function setupNewChromedriver (opts, curDeviceId, adbPort) {
+  let getPort = BlueBird.promisify(PortFinder.getPort, {context: PortFinder});
+  let chromePort = await getPort();
+  logger.error(`Using ${chromePort} for new ChromeDriver session`);
   let chromeArgs = {
-    port: opts.chromeDriverPort,
+    port: chromePort,
```

Comment:
If people are not relying on the port being 9515 (i.e., if when they don't set the port themselves they don't care what port is used) the default can be removed entirely, so it will be undefined if not set.